apps/nextjs-website/src/helpers/auth.helpers.ts

```diff
@@ -0,0 +1,4 @@
+export const passwordMatcher =
+  /(?=(.*[0-9]))(?=.*[!@#$%^&*()\\[\]{}\-_+=~`|:;"'<>,./?])(?=.*[a-z])(?=(.*[A-Z]))(?=(.*)).{8,}/;
+
+export const emailMatcher = /^[\w-\\.]+@([\w-]+\.)+[\w-]{2,4}$/;
```

Comment:
The `+` sign is not allowed during sign up.

I'm not sure at this moment we should make this validation.

Reply:
I updated the matcher to include emails with the `+` sign